libs/ledger-live-common/src/env.ts

```diff
@@ -29,6 +29,22 @@ const boolParser = (v: unknown): boolean | null | undefined => {
 const stringParser = (v: unknown): string | null | undefined =>
   typeof v === "string" ? v : undefined;
 
+type JSONValue =
+  | string
+  | number
+  | boolean
+  | { [x: string]: JSONValue }
+  | Array<JSONValue>;
```

Comment:
Can the JSON value also be `null` ?

Reply:
true, I'll fix that in the other PR dedicated to this